Sources/Target/Windows/Process.cpp

```diff
@@ -104,10 +122,52 @@ ErrorCode Process::detach() {
   return kSuccess;
 }
 
+BOOL Process::writeDebugBreakCode(uint64_t address) {
+  FARPROC exitThreadAddress =
+      GetProcAddress(GetModuleHandleA("ntdll"), "RtlExitUserThread");
```

Comment:
Is it correct that `GetModuleHandleA` cannot fail here? I think you have to have that assumption correct otherwise this code may fail. (Either in ds2, or with a segfault in the inferior if `GetProcAddress` returns `0`).